lib/grunt/utils.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
       done();
       return;
     }
-    var p = spawn('node', ['node_modules/protractor/bin/webdriver-manager', 'update']);
+    var p = spawn('node', [require.resolve('protractor/bin/webdriver-manager'), 'update']);
```

Comment:
I didn't want to complicate the PR actually, but if this is future-proof then let's do it